runtime/ts/keymgmt/webcrypto.ts

```diff
@@ -0,0 +1,276 @@
+import {DeviceKey, Key, PrivateKey, PublicKey, RecoveryKey, SessionKey, WrappedKey} from "./keys";
+import {KeyGenerator, KeyStorage} from "./manager";
+import idb, {ObjectStore} from 'idb';
+import {decode, encode} from './base64';
+import rs from 'jsrsasign';
+
+class WebCryptoStorableKey<T> {
+
+    protected key: T;
+
+    constructor(key: T) {
+        this.key = key;
+    }
+
+    algorithm(): string {
+        // @ts-ignore
```

Comment:
Add TODO on why this is here and how to potentially fix more properly.

Reply:
Removed